sparse/tpls/KokkosSparse_spmv_tpl_spec_decl.hpp

```diff
@@ -57,6 +56,7 @@ void spmv_cusparse(const KokkosKernels::Experimental::Controls& controls,
 
 #if defined(CUSPARSE_VERSION) && (10300 <= CUSPARSE_VERSION)
 
+  using entry_type = typename AMatrix::non_const_ordinal_type;
```

Comment:
This is not related to the SpGEMM changes, but when building with clang+cuda10 I got an unused local typedef warning so this fixes it.